### PR TITLE
Fixes #21843 - Connect the plugin to transifex and pull in the strings

### PR DESCRIPTION
**Pull request by bkearney**

### CODE REVIEW

File: locale/it/hammer-cli-foreman-admin.po [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+"Last-Translator: Bryan Kearney <bryan.kearney@gmail.com>, 2017\n"
+"Language-Team: Italian (https://www.transifex.com/foreman/teams/114/it/)\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Language: it\n"
+"Plural-Forms: nplurals=2; plural=(n != 1);\n"
+
+msgid "Administrative server-side tasks"
+msgstr ""
```
Comment (akofink) [inline locale/it/hammer-cli-foreman-admin.po:21]: Is this WIP, or are we expecting a lot of missing translations?

**Comment by bkearney** [comment]
This PR is good as is. Over time, when more folks translate future prs with each build will add more translations. The effect of no translation is showing english, so it fails gracefully.

**Comment by akofink** [comment]
Could you please add a description for how to test this?

**Comment by bkearney** [comment]
Here is the easiest way:

1) Go intro Transifex, and make an odd translation you can find later. Say, translate something in french to be "Jar Jar binks is my co-pilot"
2) Make a local branch
3) From the root of the file tree, run make -C locale tx-update
4) You should see the string from (1) in the po file in the locale directory.
5) If you run the tool with a frech locale, you shoud see you swanky translation.

**Comment by bkearney** [comment]
Bump!

**Comment by bkearney** [comment]
bump!

**Review by akofink — APPROVED** [review]
Works as expected. ACK Thanks @bkearney!

**Comment by akofink** [comment]
@theforeman/cli ready for merge, just need someone with access

File: locale/Makefile [hunk 2]
```diff
@@ -0,0 +1,65 @@
+#
+# Makefile for PO merging and MO generation. More info in the README.
+#
+# make all-mo (default) - generate MO files
+# make check - check translations using translate-tool
+# make tx-update - download and merge translations from Transifex
+# make clean - clean everything
+#
+DOMAIN = hammer-cli-foreman-admin
+VERSION = $(shell ruby -e 'require "rubygems";spec = Gem::Specification::load("../hammer_cli_foreman_openscap.gemspec");puts spec.version')
```
Comment (tstrachota) [inline locale/Makefile:10]:
```suggestion
VERSION = $(shell ruby -e 'require "rubygems";spec = Gem::Specification::load("../hammer_cli_foreman_admin.gemspec");puts spec.version')
```

**Comment by tstrachota** [comment]
I was about to merge this and then I found out there's "openscap" leftover in the makefile. I added a suggestion that @bkearney can merge with one click (hopefully, I never tried this feature before).

**Comment by lzap** [comment]
Thanks! This is now replaced with: https://github.com/theforeman/hammer-cli-foreman-admin/pull/10

**Pull request closed without merging**